lib/flutter_html.dart

```diff
@@ -112,6 +114,10 @@ class Html extends StatelessWidget {
   /// A function that defines what to do when a link is tapped
   final OnTap? onLinkTap;
 
+  /// A function that defines what to do when an anchor link is tapped. When this value is set,
+  /// the default anchor behaviour is overwritten.
+  final OnTap? onAnchorTap;
```

Comment:
Would it be better to use a different signature here rather than `OnTap`? I think it would be useful to provide the `AnchorKey` to the user as a parameter, and they probably don't need stuff like the `dom.Element` or attributes (but we can leave it in).

Reply:
@tneotia  Obviously, that will be a great solution.

Reply:
I think OnTap is just fine and you'll find the id in the element so that's useful to have. Also, API consistency.